src/main/java/net/neoforged/neoforge/event/EventHooks.java

```diff
@@ -445,8 +445,16 @@ public static boolean onEntityStruckByLightning(Entity entity, LightningBolt bol
         return NeoForge.EVENT_BUS.post(new EntityStruckByLightningEvent(entity, bolt)).isCanceled();
     }
 
+    /**
+     * @deprecated Use {@link #onItemUseStart(LivingEntity, ItemStack, InteractionHand, int) the hand sensitive version} as this version provides wrong hand information
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
Add the `since` part to this annotation.

Reply:
```suggestion
    @Deprecated(since = "1.21.5", forRemoval = true)
```